Letterbook.Workers.Tests/TimelineConsumerTests.cs

```diff
@@ -173,31 +173,6 @@ public async Task CanConsumeShared()
 		AuthzTimelineServiceMock.VerifyNoOtherCalls();
 	}
 
-	[Fact(DisplayName = "Should not add to timeline for malformed Shared events")]
```

Comment:
I removed this test because the changes to the message contract make this scenario essentially impossible